src/utils/useNumericInput.ts

```diff
@@ -33,6 +33,15 @@ function isValidChange(input: string): boolean {
   return inputRegex.test(cleanInput);
 }
 
+export const fixedDecimal = (number: number | string, fixedTo = 5): string => {
```

Comment:
https://github.com/Badger-Finance/v2-ui/blob/development/src/mobx/model/tokens/token-balance.ts

fix this file, not creating a separate method.